src/CodeGen_LLVM.cpp

```diff
@@ -2891,18 +2895,28 @@ void CodeGen_LLVM::visit(const Call *op) {
             }
             item += 1;
         } while (item < count);
-    } else if (op->is_intrinsic(Call::load_struct_member)) {
+    } else if (op->is_intrinsic(Call::load_typed_struct_member)) {
+        // Given an instance of a typed_struct, a definition of a typed_struct,
+        // and the index of a slot, load the value of that slot.
+        //
+        // An instance of a typed_struct is an Expr of Handle() type that has been
+        // created by a call to make_typed_struct.
+        //
+        // An definiton of a typed_struct is an Expr of Handle() type that has been
```

Comment:
Ditto comments on version in CodeGen_C.cpp . I find this method of documenting intrinsics highly dubious precisely for the reason that it is duplicated and thus a comment this long is basically unhelpful. Probably worth a TODO and an issue to move all intrinsics documentation into a header file with the intrinsics themselves.

Reply:
https://github.com/halide/Halide/issues/6468